test/parallel/test-dgram-multicast-set-interface.js

```diff
@@ -73,6 +73,20 @@ const dgram = require('dgram');
 }
 
 {
+  try {
+    const socket = dgram.createSocket('udp6');
+
+    socket.bind(0);
+    socket.on('listening', common.mustCall(() => {
+      socket.close();
+    }));
+  } catch (e) {
+    common.skip(`IPv6 sockets are missing or incompatible: ${e}`);
```

Comment:
There is `common.hasIPv6`, which lets you detect if IPv6 is supported before trying to do something like this.